contracts/CellarPoolShare.sol

```diff
@@ -396,16 +406,20 @@ contract CellarPoolShare is ICellarPoolShare, BlockLock {
 
         balance0 = IERC20(token0).balanceOf(address(this));
         balance1 = IERC20(token1).balanceOf(address(this));
-        _addLiquidity(
-            CellarAddParams({
-                amount0Desired: balance0,
-                amount1Desired: balance1,
-                amount0Min: 0,
-                amount1Min: 0,
-                recipient: address(this),
-                deadline: type(uint256).max
-            })
-        );
+        (inAmount0, inAmount1, , ) =
```

Comment:
not sure if this is allowed? it was initialized in the previous `_addLiquidity` call